certbot-nginx/certbot_nginx/tls_sni_01.py

```diff
@@ -51,6 +51,9 @@ def perform(self):
         default_addr = "{0} ssl".format(
             self.configurator.config.tls_sni_01_port)
 
+        ipv6info = self.configurator.ipv6_info(
```

Comment:
can you name the resulting booleans for clarity?

Reply:
Yeah, improves readability quite a bit, done.